pubsub/topics/main.go

```diff
@@ -193,6 +193,26 @@ func publishThatScales(client *pubsub.Client, topic string, n int) error {
 	// [END pubsub_publish_with_error_handling_that_scales]
 }
 
+func publishCustomAttributes(client *pubsub.Client, topic string) error {
+	ctx := context.Background()
+	// [START pubsub_publish_custom_attributes]
+	t := client.Topic(topic)
+	result := t.Publish(ctx, &pubsub.Message{
+		// data must be a ByteString
```

Comment:
```suggestion
		// Data must be a ByteString.
```

Reply:
in fact, i'd just cut this comment (and the one above where it was copied from.

ByteString doesn't mean anything in Go. (`[]byte` is a very common type)